content/docs/terraform-tools.mdx

```diff
@@ -0,0 +1,70 @@
+
+
+## Automate Terraform
+
+|Name | Maintainer |Description| Tutorial |
+|-----|------------|-----------|----------|
+| [`terraform-exec`](https://github.com/hashicorp/terraform-exec) | HashiCorp | Use the Go module to construct and run CLI commands. Refer to the `terraform-json` repository (below) for documentation about the return values. |  |
+| [`terraform-json`](https://github.com/hashicorp/terraform-json) | HashiCorp | Use the data types and helper functions in this repository to work with JSON-formatted Terraform output.|  |
+| [Terraform Orb](https://circleci.com/developer/orbs/orb/circleci/terraform) | Third Party | Deploy your Terraform infrastructure via a CircleCI pipeline. CircleCI Orbs let you package, ship, and re-use configurations. | [Deploy Terraform infrastructure with CircleCI](https://learn.hashicorp.com/tutorials/terraform/circle-ci) |
+|[GitHub Actions for Terraform](https://github.com/hashicorp/setup-terraform) | Third Party | Automate Terraform in your GitHub Actions workflow. After you have used the action, subsequent steps in the same job can run Terraform commands using the GitHub Actions run syntax. | [Automate Terraform with GitHub Actions](https://learn.hashicorp.com/tutorials/terraform/github-actions) |
+
+## Security and Compliance
+
+|Name | Maintainer |Description| Tutorial |
+|-----|------------|-----------|----------|
+| [Terraform Cloud for Splunk](/docs/cloud/integrations/splunk/index.html) | HashiCorp | Pull audit logs from Terraform Cloud into Splunk to identify the most active policies, understand significant changes in resource operations, and filter actions by specific users within your organization. | |
+| [Sentinel](https://www.terraform.io/docs/cloud/sentinel/index.html) | HashiCorp |  Write and enforce fine-grained, logic-based policies for your infrastructure. Sentinel is an embedded policy-as-code framework integrated with the HashiCorp Enterprise products. | [Enforce Policy with Sentinel](https://learn.hashicorp.com/collections/terraform/policy) |
+
+
+## Develop Terraform Providers
+
+|Name | Maintainer |Description| Tutorial |
+|-----|------------|-----------|----------|
+| [Plugin SDK](/docs/extend/sdkv2-intro.html) | HashiCorp | Develop your own Terraform providers. | [Setup and Implement Read](https://learn.hashicorp.com/tutorials/terraform/provider-setup?in=terraform/providers) |
+|[Plugin Framework](/docs/plugin/framework/index.html) | HashiCorp | Develop your own Terraform providers. The framework provides improvements and additional features beyond what is available in the Plugin SDK. | [Implement Create and Read with the Terraform Plugin Framework](https://learn.hashicorp.com/tutorials/terraform/plugin-framework-create?in=terraform/providers) |
```

Comment:
This doesn't give the user much guidance as to where to go to first if they want to create a new provider.

Reply:
that's fair - this is a complex thing actually that I struggled with BUT I'm going to push up a tiny change to indicate that this ia "A new SDK to develop your own Terraform providers." This way it's clear from the first sentence that this is the most recent option, differentiating it a bit from the one above.